api/src/main/java/io/opentelemetry/trace/TracingContextUtils.java

```diff
@@ -30,7 +30,21 @@
 @Immutable
 public final class TracingContextUtils {
   private static final Context.Key<Span> CONTEXT_SPAN_KEY =
-      Context.<Span>key("opentelemetry-trace-span-key");
+      Context.<Span>keyWithDefault("opentelemetry-trace-span-key", DefaultSpan.getInvalid());
```

Comment:
Why this change, you just changed it to not use default?

Reply:
Same here if we keep the  previous code it should just work.

Reply:
Ok, lets do that then. I will then iterate with a new PR to -possibly- change things.